wordcamp.dev/provision/vvv-init.sh

```diff
@@ -62,6 +62,25 @@ else
 
 	git -C $SITE_DIR/wp-content/plugins/camptix pull origin master
 
+	wme_noroot wp plugin install supportflow            --path=$SITE_DIR/wordpress
+	wme_noroot wp plugin install polldaddy              --path=$SITE_DIR/wordpress
+	wme_noroot wp plugin install liveblog               --path=$SITE_DIR/wordpress
+	wme_noroot wp plugin install camptix-trustpay       --path=$SITE_DIR/wordpress
+	wme_noroot wp plugin install camptix-trustcard      --path=$SITE_DIR/wordpress
+	wme_noroot wp plugin install camptix-mercadopago    --path=$SITE_DIR/wordpress
+	wme_noroot wp plugin install camptix-kdcpay-gateway --path=$SITE_DIR/wordpress
+	wme_noroot wp plugin install wp-super-cache         --path=$SITE_DIR/wordpress
+	wme_noroot wp plugin install custom-content-width   --path=$SITE_DIR/wordpress
+
```

Comment:
@grappler Instead of installing each of these on a separate line, you can add them to the `WPCLI_PLUGINS` array at the top of the script and then they will all get installed in one line [here](https://github.com/WordPress/meta-environment/blob/master/wordcamp.dev/provision/vvv-init.sh#L40).